.github/workflows/process-and-transfer.yml

```
@@ -33,6 +33,11 @@ jobs:
             config_file: data_management/hkh_watermaps.json
           - environment: Wildfires
             config_file: data_management/nm_wildfires.json
+          - environment: USDA
+            config_file: data_management/usda_china2021.json
+            config_file: data_management/usda_china2022.json
+            config_file: data_management/usda_madagascar.json
+            config_file: data_management/usda_pakistan.json
```

```suggestion
          - environment: USDA
            config_file: data_management/usda_china2021.json
          - environment: USDA
            config_file: data_management/usda_china2022.json
          - environment: USDA
            config_file: data_management/usda_madagascar.json
          - environment: USDA
            config_file: data_management/usda_pakistan.json
```
@hjkristenson I believe you can write it like this. I think you can deploy to the same environment from multiple jobs.

---

Great. I considered doing that, but thought there might be a more efficient way.